### DISCUSSION TITLE
How to enable back disabled device

### DISCUSSION THREAD

**Discussion by Jakhontoviy** [make-all/tuya-local#486] [Q&A]
Hi all,

Just to test the feature, I've used disable device for my switch. Now I cann't find a button to enable it back. Could you please halp me.

BTW, thanks for the great product! It is in 1000 times better than local tuya :)

**Comment by make-all** [comment]
On the "Devices" tab of Settings / Devices & Services, you should see a button at top right that says "x devices not shown [CLEAR]".  Click on that to show your disabled devices, then search for it in the list.  Open the device details and click the Edit button at top right, you should see a switch to enable it again.

**Reply by Jakhontoviy** [reply]
Thanks for your help, I found a button but unfortuantely it disable, so I cann't enable it back from there:

![image](https://user-images.githubusercontent.com/79797608/222890270-9952e1ab-b1ea-4bb3-873a-d7defbc8b7fb.png)

**Reply by Jakhontoviy** [reply]
Finally I found the button. It was under Integrations tab instead of devices:

![image](https://user-images.githubusercontent.com/79797608/222892794-1a81b82e-b6b1-4df6-bd15-7bc1f17a851b.png)

Thanks a lot!
